src/plugins/intel_cpu/src/config.h

```diff
@@ -62,7 +62,15 @@ struct Config {
     size_t rtCacheCapacity = 0ul;
 #endif
     ov::threading::IStreamsExecutor::Config streamExecutorConfig;
+    int streams = 1;
+    bool streamsChanged = false;
+    int threads = 0;
+    int threadsPerStream = 0;
+    bool cpuReservation = false;
+    std::vector<std::vector<int>> streamsInfoTable;
```

Comment:
Why I need to store streams info table here. I except it to be initialized and stored inside `streamExecutorConfig`

Reply:
Because the calculation of streams info table is in CPU plugin. Now we unified to set properties by calling constructor `IStreamsExecutor::Config(.., .., streamsInfoTable, ..)` in `CompiledModel::CompiledModel`, so I store this property in config which is passed to `CompiledModel()` and then set `streamsInfoTable` to `IStreamsExecutor::Config()`.